packages/elements/src/heatmap/helpers/text.ts

```diff
@@ -95,22 +95,12 @@ const getMaximumTextWidth = (ctx: CanvasRenderingContext2D, cells: HeatmapCell[]
 };
 
 /**
- * Calculate max length text for render Axis size
- * @param labels array string text for calculate
- * @returns max length text
+ * Get label with maximum length
+ * @param labels array string labels
+ * @returns Label with maximum length
  */
-const getMaximumLabelTextWidth = (labels: string[]): string => {
-  let maxLengthText = '';
-
-  for (let i = 0; i < labels.length; i++) {
-    if (maxLengthText.length >= labels[i].length) {
-      maxLengthText = maxLengthText;
-    }
-    else {
-      maxLengthText = labels[i];
-    }
-  }
-  return maxLengthText;
+const getMaximumLabelWidth = (labels: string[]): string => {
+  return labels.reduce((previousLabel, currentLabel) => previousLabel.length > currentLabel.length ? previousLabel : currentLabel);
 };
 
-export { getResponsiveFontSize, getMaximumTextWidth, getMaximumLabelTextWidth, MIN_FONT_SIZE, MAX_FONT_SIZE };
+export { getResponsiveFontSize, getMaximumTextWidth, getMaximumLabelWidth, MIN_FONT_SIZE, MAX_FONT_SIZE };
```

Comment:
What I actually don't like about the method name is that it ends with `Width` so I would assume that this method returns width but actually, it returns the longest label.

You can see that `getMaximumTextWidth` and `getMaximumLabelWidth` have similar name while the first one is returning width (which makes sense for its name).

Reply:
That's true. I'll go for `getLongestLabel()` instead.

Reply:
Actually we might not even need to extract this method at all. I can just place this line onto the code itself.

Reply:
I like that :) I want to push one line comment and I will approve this PR